plugin/src/__snapshots__/theme.spec.ts.snap

```diff
@@ -108,17 +108,6 @@ Object {
       "M400": "#d57287",
       "M600": "#b23a5d",
     },
-    "product": Object {
-      "chat": "#f79a3e",
-      "connect": "#eb6651",
-      "explore": "#30aabc",
-      "gather": "#e7afa2",
-      "guide": "#eb4962",
-      "message": "#37b8af",
-      "sell": "#d4ae5e",
-      "support": "#78a300",
-      "talk": "#efc93d",
-    },
```

Comment:
The removal of this key might warrant some discussion. We never want folks to use these product color values for theming (which is why they are removed from the `DEFAULT_THEME` in `@zendeskgarden/react-theming`). Furthermore, I'm considering this to be a patch – as the initial addition as design tokens could be considered unintentional – but I could be convinced to re-characterize this PR as a "BREAKING CHANGE".

Reply:
I would like to discuss this removal more.

> We never want folks to use these product color values for theming

I agree with this in relation to dynamic theming in `react-components`, but these values could be used often throughout a codebase which uses `tailwindcss`. If we were to remove these products would hardcode the values and no longer receive updates as values are updated.

Reply:
Confirmed by design: product colors are only for product and only used for 1️⃣ purpose – the logo. Never to be used in the context of design tokens.